package.json

```diff
@@ -6,12 +6,15 @@
     "license": "MIT",
     "directories": {"lib": "./lib"},
     "scripts": {
+      "prepublish": "npm run-script build",
+      "build": "rollup --banner \"`cat src/banner.js`\" --format umd -n CodeMirror src/codemirror.js -o lib/codemirror.js || true",
```

Comment:
Is the `|| true` temporary? The cruft npm adds when a script fails is pretty awful, but I think in this case we'd want to preserver the return value.